### PR TITLE
Update references to py-rcsb_exdb_assets from development to master

### PR DESCRIPTION
**Pull request by brindakv**

### CODE REVIEW

**Comment by aliciaaevans** [comment]
Changes look good to me. It's just missing an update in the header comments of `config/dbload-setup-example.yml`.

**Comment by brindakv** [comment]
Thanks @aliciaaevans. I updated the header comment.

**Review by aliciaaevans — APPROVED** [review]

File: config/dbload-setup-example.yml [hunk 1]
```diff
@@ -121,27 +122,27 @@ site_info_configuration:
...
+  PDBX_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/mmcif_pdbx_v5_next.dic
+  RCSB_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/dist/rcsb_mmcif_ext.dic
+  RCSB_COMP_MODEL_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/dist/rcsb_mmcif_comp_model_ext.dic
+  FLR_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/flr-extension.dic
+  VRPT_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/vrpt_mmcif_ext.dic
+  VRPT_DICT_MAPPING_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/vrpt_dictmap.json
+  IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/ihm-extension.dic
+  MA_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/mmcif_ma.dic
```
Comment (piehld) [inline config/dbload-setup-example.yml]: I think this should be `mmcif_ma_ext.dic`?

File: config/dbload-setup-example.yml [hunk 2]
```diff
@@ -121,27 +122,27 @@ site_info_configuration:
...
   ## Previously pointed to 'dictionary_files/reference', but point to authoritative source instead to ensure most recent version is used
   # IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/ihmwg/IHM-dictionary/master/ihm-extension.dic
   # MA_DICT_LOCATOR: https://raw.githubusercontent.com/ihmwg/MA-dictionary/master/mmcif_ma.dic
```
Comment (piehld) [inline config/dbload-setup-example.yml]: Maybe delete these comments?

File: config/dbload-setup-example.yml [hunk 3]
```diff
@@ -272,26 +273,26 @@ site_info_remote_configuration:
...
+  PDBX_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/mmcif_pdbx_v5_next.dic
+  RCSB_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/dist/rcsb_mmcif_ext.dic
+  RCSB_COMP_MODEL_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/dist/rcsb_mmcif_comp_model_ext.dic
+  FLR_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/flr-extension.dic
+  VRPT_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/vrpt_mmcif_ext.dic
+  VRPT_DICT_MAPPING_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/vrpt_dictmap.json
+  IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/ihm-extension.dic
+  MA_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/mmcif_ma.dic
```
Comment (piehld) [inline config/dbload-setup-example.yml]: Also here

**Review by piehld — COMMENTED** [review]
Just one change I found, but otherwise looks good. Thanks!

File: config/dbload-setup-example.yml (hunk 3 above)
Comment (brindakv) [reply config/dbload-setup-example.yml]: Good catch @piehld. Fixed.

**Review by piehld — APPROVED** [review]

**Pull request merged**